rmw_dds_common/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ ament_export_dependencies(rcpputils)
 ament_export_dependencies(rcutils)
 ament_export_dependencies(rmw)
 ament_export_dependencies(rosidl_cmake)
+ament_export_dependencies(rosidl_runtime_cpp)
```

Comment:
Isn't this the same line as below?

Reply:
all this 4 lines are repeated. should I removed all of them? or open another PR?

Reply:
Either is fine with me.

Reply:
fixed https://github.com/ros2/rmw_dds_common/pull/10/commits/def6296f2afa229e019ea41c06f5d2a5156589a2